code/datums/components/fantasy/suffixes.dm

```diff
@@ -0,0 +1,95 @@
+/datum/fantasy_affix/cosmetic_suffixes
+	placement = AFFIX_SUFFIX
+	alignment = AFFIX_GOOD | AFFIX_EVIL
+
+	var/list/goodSuffixes
+	var/list/badSuffixes
+
+/datum/fantasy_affix/cosmetic_suffixes/New()
+	goodSuffixes = list(
+		"dexterity",
+		"constitution",
+		"intelligence",
+		"wisdom", 
+		"charisma",
```

Comment:
+movement speed, +health and maxhealth (maybe even regen?), +sapience (like the chaplain's possessed blade), +various HUDs, +access to that cult conversion spell that runic golems get access to (the one that turns simplemobs to your side), respectively